packages/core/src/createActor.ts

```diff
@@ -674,6 +676,9 @@ export class Actor<TLogic extends AnyActorLogic>
     return this.logic.getPersistedSnapshot(this._snapshot, options);
   }
 
+  /**
+   * @internal
+   */
```

Comment:
If you mark this as internal it will break people relying on this. This one exists for a reason here.
```suggestion
```